dump/dbus_util.cpp

```diff
@@ -186,7 +186,17 @@ const std::vector<std::string>
             "/xyz/openbmc_project/object_mapper",
             "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths");
         const int32_t depth = 0;
-        std::vector<std::string> intf{entryIntf};
+        std::vector<std::string> intf;
+        if (entryIntf.find("System") != std::string::npos)
+        {
+            intf.push_back("com.ibm.Dump.Entry.SBE");
```

Comment:
Suggestion: You could assign multiple elements like this too, as we are building the list from scratch and not pushing it to existing list.
intf.assign({
    "com.ibm.Dump.Entry.SBE",
    "com.ibm.Dump.Entry.Hostboot",
    "com.ibm.Dump.Entry.Hardware"
});

Reply:
+1